policy-manager/js/MainController.js

```diff
@@ -272,10 +272,12 @@ Ext.define('Ung.apps.policymanager.MainController', {
             }
             Ext.getStore('policiestree').build();
         }, function(ex) {
-            if(Util.isDestroyed(view)){
-                view.setLoading(true);
+            view.setLoading(false);
```

Comment:
I needed to do `setLoading(false)` or left panel was showing the loader and no action to the policies were allowed. Need to go back to the Apps page and come back again to do any action.
![image](https://github.com/untangle/ngfw_src/assets/154527616/79b1121d-5bba-4b32-98f1-ff8f9e79ed9b)